### ISSUE TITLE
Separate Development Tools from the Development Guide

### ISSUE THREAD

**Issue by mcdonnnj** [cisagov/development-guide#33]
# 🚀 Feature Proposal

Separate the Development Tools currently stored in [project_setup/scripts](https://github.com/cisagov/development-guide/tree/develop/project_setup/scripts) into their own repository as a Python package.

## Motivation

@hillaryj created an issue to add testing to the scripts in the above mentioned directory in #31 , but I believe it would be better to separate tooling from documentation. 

## Example

One would be able to install the development tools as a Python package a la `pip install git+https://github.com/cisagov/development-tools.git` and the scripts would be available in the environment.

## Pitch

As it stands it is difficult to add testing and keep Python related skeleton updates current in this project because:
1. It descends from [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic) instead of [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library)
1. It is a repository trying to store both documentation about our development process and the tools that are used

Moving the tooling into its own repository would allow it to be configured as a typical Python package, which would allow for our CI/CD pipeline to be implemented as it typically would for such a project. This will make it easier to add new tooling, eliminate duplicate code between tools, and ensure that testing is done automatically.

**Comment by jsf9k** [comment]
I wholeheartedly support this change.

**Comment by hillaryj** [comment]
I've created a new `skeleton-python-library`-based repo at https://github.com/cisagov/development-tools

**Comment by jsf9k** [comment]
> I've created a new `skeleton-python-library`-based repo at https://github.com/cisagov/development-tools

I don't think this was the intent.  We wanted to move _each_ development tool to _its own_ repo so we can apply all our CI machinery to it.
